lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1621,6 +1624,17 @@ public function getCreateTableSQL(Table $table, $createFlags = self::CREATE_INDE
         return array_merge($sql, $columnSql);
     }
 
+    public function getCommentOnTableSQL(string $tableName, ?string $comment) : string
+    {
+        $tableName = new Identifier($tableName);
+
+        return sprintf(
+            'COMMENT ON TABLE %s IS %s',
+            $tableName->getQuotedName($this),
+            $this->quoteStringLiteral((string) $comment)
```

Comment:
It's questionable that comments are nullable. There doesn't seem to be a difference between NULL and an empty string in this case. Would it make sense to not allow NULL in the first place to avoid a potential BC break? It may become irrelevant if the method is not exposed as `public`.

Reply:
Completely agree, but same thing here. I simply copied the `getCommentOnColumnSQL` method behaviour. In `getCommentOnColumnSQL`, comments are already a nullable string. Shall I change this or keep this as-is for consistency?

Reply:
Let's keep it consistent with the rest of the methods for now.